django/contrib/admin/templates/admin/change_form.html

```diff
@@ -73,15 +73,39 @@
             $(document).ready(function() {
```

Comment:
I can't see how we can pass parameters from django on an external js but we can eventually store the django knowledge in the dom (e.g. a data-django_changeformadd attribute) and get that at runtime. While i agree that inline js like this is a bit gross and usually like this approach am pretty sure it would be quite a surprise if you want to override the template and missing that dom attribute that breaks the js.